### PR TITLE
EZP-30128: [Follow-up] Fix Sub-items delete/trash actions'  messages

### PR DESCRIPTION
**Pull request by tischsoic**
JIRA: https://jira.ez.no/browse/EZP-30128

LhSklY9I7l

### CODE REVIEW

File: src/modules/sub-items/sub.items.module.js [hunk 1]
```diff
@@ -469,7 +469,13 @@ export default class SubItemsModule extends Component {
                 ),
             };
 
-            this.handleBulkOperationFailedNotification(selectedItems, notMovedLocations, modalTableTitle, notificationMessage, rawPlaceholdersMap);
+            this.handleBulkOperationFailedNotification(
```
Comment (tischsoic) [inline src/modules/sub-items/sub.items.module.js:472]: Only Prettier here.

File: src/modules/sub-items/sub.items.module.js [hunk 2]
```diff
@@ -636,15 +642,19 @@ export default class SubItemsModule extends Component {
             };
         });
 
-        window.eZ.helpers.notification.showWarningNotification(notificationMessage, (notificationNode) => {
-            const showModalBtn = notificationNode.querySelector('.ez-notification-btn--show-modal');
+        window.eZ.helpers.notification.showWarningNotification(
```
Comment (tischsoic) [inline src/modules/sub-items/sub.items.module.js:709]: Only Prettier here.

File: Resources/translations/sub_items.en.xliff [hunk 3]
```diff
@@ -31,10 +31,20 @@
         <target state="new">Delete</target>
         <note>key: bulk_delete.popup.confirm</note>
       </trans-unit>
-      <trans-unit id="02d93b03435046c6bcf08ce15ae7970f7ff8f746" resname="bulk_delete.popup.message">
-        <source>Are you sure you want to delete the selected content item(s)?</source>
-        <target>Are you sure you want to delete the selected content item(s)?</target>
-        <note>key: bulk_delete.popup.message</note>
+      <trans-unit id="e98d09770399a6f52a15068a0573a18fc2b0b60d" resname="bulk_delete.popup.message.nonusers">
+        <source>Are you sure you want to send to the trash the selected content item(s)?</source>
```
Comment (SylvainGuittard) [inline Resources/translations/sub_items.en.xliff]:
```suggestion
        <source>Are you sure you want to send to trash the selected content item(s)?</source>
```

File: Resources/translations/sub_items.en.xliff [hunk 4]
```diff
@@ -31,10 +31,20 @@
...
+        <target state="new">Are you sure you want to send to the trash the selected content item(s)?</target>
```
Comment (SylvainGuittard) [inline Resources/translations/sub_items.en.xliff]:
```suggestion
        <target state="new">Are you sure you want to send to trash the selected content item(s)?</target>
```

**Review by SylvainGuittard — CHANGES_REQUESTED** [review]

File: Resources/translations/sub_items.en.xliff [hunk 5]
```diff
@@ -31,15 +51,35 @@
...
-        <source>Are you sure you want to delete the selected content item(s)?</source>
-        <target>Are you sure you want to delete the selected content item(s)?</target>
-        <note>key: bulk_delete.popup.message</note>
-      </trans-unit>
-      <trans-unit id="684c07e2248d815e6eb6bc3f92d3cd49c6952cbe" resname="bulk_delete.success.message">
-        <source>The selected content item(s) have been deleted</source>
-        <target state="new">The selected content item(s) have been deleted</target>
-        <note>key: bulk_delete.success.message</note>
+      <trans-unit id="e98d09770399a6f52a15068a0573a18fc2b0b60d" resname="bulk_delete.popup.message.nonusers">
+        <source>Are you sure you want to send to the trash the selected content item(s)?</source>
```
Comment (DominikaK) [inline Resources/translations/sub_items.en.xliff]:
```suggestion
        <source>Are you sure you want to send the selected content item(s) to trash?</source>
```

File: Resources/translations/sub_items.en.xliff [hunk 6]
```diff
@@ -31,15 +51,35 @@
...
+        <target state="new">Are you sure you want to send to the trash the selected content item(s)?</target>
```
Comment (DominikaK) [inline Resources/translations/sub_items.en.xliff]:
```suggestion
        <target state="new">Are you sure you want to send the selected content item(s) to trash?</target>
```

**Review by dew326 — APPROVED** [review]

**Comment by lserwatka** [comment]
This is candidate to 2.5.3 @barbaragr we need to merge it today if possible.

**Comment by barbaragr** [comment]
@lserwatka sure, on my way 🚀

**Review by dew326 — APPROVED** [review]

**Review by barbaragr — APPROVED** [review]

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by tischsoic** [comment]
Merged up:
- master: https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/0f84f0a96aacc9df2828f7af089cf204f82d91e9

**Pull request merged**